satisfactory-accounting/src/database.rs

```diff
@@ -91,6 +98,9 @@ impl DatabaseVersion {
                 "This is the first version of the Satisfactory Accounting database \
                 released after the U6 update. Please report missing/incorrect recipes!"
             }
+            DatabaseVersion::U7(U7Subversion::Final) => {
+                "This is the final version of the database released for U7."
```

Comment:
This is the first time we're including a database for U7, so I would call it Beta or Initial for now, that way if we later discover a mistake in the database and have to release a new version, we don't have to rename this subversion, we just create a new U7Subversion for the new release.

I prefer this approach because I'd rather things be consistent through the git revision history rather than something where every time a new revision comes along the 'newly added files' are actually a copy of the old revision rather than actually the new revision.